base/special/trig.jl

```diff
@@ -200,6 +200,25 @@ ACOS_CORRECT_LOWWORD(::Type{Float64}, x) = reinterpret(Float64, (reinterpret(UIn
 
 @noinline acos_domain_error(x) = throw(DomainError(x, "acos(x) not defined for |x|>1"))
 function acos(x::T) where T <: Union{Float32, Float64}
+    # Method :
```

Comment:
It could be moved. It should be possible to infer it, but some things aren't too well commented in the actual code. The solution is of course... to move the comments to the appropriate places.